Makefile

```diff
@@ -2,7 +2,7 @@
 # MAINTAINER: Steve Sloka <steve@stevesloka.com>
 # If you update this image please bump the tag value before pushing.
 
-TAG = 1.9
+TAG = 1.10
```

Comment:
Please don't change versions in this, that is a separate process.

Reply:
Fixed!